scripts/integration.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -ex
+set -o pipefail
+
+if [ ! -f genfiles/server.key ]; then
+	./scripts/prepare_server.sh -f
+fi
+
+docker-compose -f docker-compose.yml -f docker-compose.prod.yml up -d
+trap "docker-compose down" INT EXIT
+
+set +e
+wget -t 60 --spider --retry-connrefused --waitretry=1  0.0.0.0:443 || sleep 2
+wget -t 60 --spider --retry-on-http-error=404  1 0.0.0.0:443/v1/directories/default
```

Comment:
nit: Could prefix with "https://" rather than specifying port 443.
```suggestion
wget -t 60 --spider --retry-on-http-error=404  1 https://0.0.0.0/v1/directories/default
```